dags/hra.py

```diff
@@ -16,6 +17,24 @@
     schedule_interval='@monthly'
 )
 
+
+def _check_hra_api_availability():
+    now = datetime.datetime.now()
+
+    is_mon_wed_or_thrusday = (now.weekday() in [0, 2, 3])
+    is_after_6am = (now.time() >= datetime.time(6, 0))
+    is_before_8am = (now.time() < datetime.time(8, 0))
+    is_between_6am_and_8am = (is_after_6am and is_before_8am)
```

Comment:
This thing will always make me laugh. I wonder if there's someone there manually turning a knob.